inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -361,34 +313,6 @@ def prizes2marc(self, key, value):
     }
 
 
-@hepnames.over('_public_note', '^680..')
-@utils.for_each_value
-def _public_note(self, key, value):
-    return value.get('i')
-
-
-@hepnames2marc.over('680', '^_public_note$')
-@utils.for_each_value
-def _public_note2marc(self, key, value):
-    return {
-        'i': value
-    }
-
-
-@hepnames.over('_curators_note', '^667..')
```

Comment:
This is a non public note. We should also add it as a meta-field to all schemas. They are useful for cataloguers to write down notes.